code/src/reducers/todos.js

```diff
@@ -0,0 +1,68 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  items: [
+    {
+      id: 1,
+      text: "Complete Project",
+      isComplete: false,
+      createdDate: 1606383658465,
```

Comment:
Since this is hardcoded it will remain this date. 
If you wanted it to appear on the same date as the user opens the app I think you could have deleted it. Making the code like this

```
       id: 1,
       text: "Complete Project",
       isComplete: false,
```